server/psql/models.js

```diff
@@ -0,0 +1,21 @@
+const db = require('./index.js');
+
+module.exports = {
```

Comment:
This looks great! Only question, mostly out of curiosity: why defining the function inside of module.exports? I guess I'm asking because I've only ever seen the functions defined outside of module.exports, and then being exported by name only. Is it for cleaner/less code?

It's a tiny thing so I'll go ahead and merge.

Reply:
Lol I don't know why I didn't notice Han already merged this 🙃

Reply:
Hi @jabrake, great question! Honestly, I am just trying out different styles to see which one works best, haha. I've seen this style used by Hack Reactor before though especially in earlier sprints. If it's just a bunch of methods being exported, I just found it slightly cleaner for them to be wrapped and exported in a group like this, although this is purely subjective. Please let me know if you found any additional info on this styling though!